windows.versions

```diff
@@ -12,6 +12,18 @@
       "latest": true,
       "stable": false
     },
+    {
```

Comment:
remember to run the new Makefile target for validating this file which is in our runbook now. 

@matthewfala @rawahars should we make that target run in the release target? that way we fail fast. 

```
make validate-version-file-format
```

Reply:
Ran the script. It works.

Reply:
what do you think about just making it part of the `release` target so we already run it a fail fast?

Reply:
That's a good idea. I can submit a pr for it.

Reply:
```
.PHONY: validate-version-file-format
validate-version-file-format:
	jq -e . windows.versions && true || false
	jq -e . linux.version && true || false
```
Actually, it's not clear what the point of this script is,

Is it supposed to fail if the following is not in JSON format?
```
{
  "linux": {
    "version": "2.31.1",
    "build": "1",
    "fluent-bit": "1.9.10",
    "kinesis-plugin": "v1.10.1",
    "firehose-plugin": "v1.7.1",
    "cloudwatch-plugin": "v1.9.1"
  }
}
```

the make validate-version-file-format does not, if that is the case.

Meanwhile it appears that the build linux plugins does fail if the version document is not JSON regardless of whether the validate-version-file-format command is run or not

```
./scripts/build_plugins.sh \
        --KINESIS_PLUGIN_CLONE_URL= \
        --KINESIS_PLUGIN_TAG= \
        --KINESIS_PLUGIN_BRANCH= \
        --FIREHOSE_PLUGIN_CLONE_URL= \
        --FIREHOSE_PLUGIN_TAG= \
        --FIREHOSE_PLUGIN_BRANCH= \
        --CLOUDWATCH_PLUGIN_CLONE_URL= \
        --CLOUDWATCH_PLUGIN_TAG= \
        --CLOUDWATCH_PLUGIN_BRANCH=
latest version not found for linux
make: *** [linux-plugins] Error 1
```

Our runbook specifies:
The above changes go in the `linux.version` and `windows.versions` file. Run `make validate-version-file-format` to check that the update was successful. If formatted incorrectly, it will lead to a release pipeline failure. 

If the JSON file is not formatted correctly, it won't be printed out, so that may be a way to manually check for issues.

Reply:
@PettitWesley I think it would be. great if we add it to the `release` target.

@matthewfala Yes that is correct. The make target would fail if the json files for Windows and Linux are not formatted properly. It would be a fast fail to ensure that our Pipelines do not fail due to invalid JSON. 

I am not clear what you mean when you say `the make validate-version-file-format does not, if that is the case.`

For example-
```
{
  "linux": {
    "version": "2.31.1",
    "build": "1",
    "fluent-bit": "1.9.10",
    "kinesis-plugin": "v1.10.1",
    "firehose-plugin": "v1.7.1",
    "cloudwatch-plugin": "v1.9.1"
  }
}
```
This would succeed since the format of json is correct.
```
{
  "linux": {
    "version": "2.31.1",
    "build": "1",
    "fluent-bit": "1.9.10",
    "kinesis-plugin": "v1.10.1",
    "firehose-plugin": "v1.7.1",
    "cloudwatch-plugin": "v1.9.1",
  }
}
```
This would fail with the following error-
```
jq -e . linux.version && true || false
parse error: Expected another key-value pair at line 9, column 3
make: *** [validate-version-file-format] Error 1
```
This is because the above JSON is not formatted properly and has an extra comma at `    "cloudwatch-plugin": "v1.9.1",`. Such errors can be easily identified before we commit the changes leading to pipeline failures.